db/warehouse/migrate/20240731145826_validate_current_living_situation_verified_by_project_id.rb

```diff
@@ -0,0 +1,10 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class ValidateCurrentLivingSituationVerifiedByProjectId < ActiveRecord::Migration[7.0]
+  def change
+    validate_foreign_key :CurrentLivingSituation, :Project
+  end
+end
```

Comment:
I added the separate migration due this warning from `strong_migrations`:

```
=== Dangerous operation detected #strong_migrations ===

Adding a foreign key blocks writes on both tables. Instead,
add the foreign key without validating existing rows,
then validate them in a separate migration.

class AddVerifiedByProjectIdToCurrentLivingSituation < ActiveRecord::Migration[7.0]
  def change
    add_foreign_key :CurrentLivingSituation, :Project, column: :verified_by_project_id, validate: false
  end
end

class ValidateAddVerifiedByProjectIdToCurrentLivingSituation < ActiveRecord::Migration[7.0]
  def change
    validate_foreign_key :CurrentLivingSituation, :Project
  end
end

/app/db/warehouse/migrate/20240731144633_add_verified_by_project_id_to_current_living_situation.rb:4:in `change'
```

I guess it was just my instinct that it would be better to validate the foreign key, but we could leave it as `validate: false` without following up to validate in a separate migration. Is that our common practice?